modules/mvt/src/mvt-loader.js

```diff
@@ -22,7 +22,7 @@ export const MVTWorkerLoader = {
     'application/x-protobuf'
     // 'application/octet-stream'
   ],
-  worker: true,
+  worker: false,
```

Comment:
This was just for testing? You can use `_workerType: 'test'` to build against local workers created by `yarn build`.

Reply:
Good spot, this was indeed just testing. Will fix